coda-prototype/UIUtils.js

```diff
@@ -491,7 +490,55 @@ var UIUtils = (function() {
         isScrolledToTop: function(tableContainer) {
             return UIUtils.checkVisible($(".message-row").first()[0]);
             //return $("#message-panel").scrollTop() === 0;
-        }
+        },
+
+        /**
+         * Displays the given message in a green banner at the top of the screen.
+         * Success banners are automatically dismissed after a few seconds, and the contents automatically cleared.
+         * @param message HTML to display in the banner.
+         */
+        displayAlertAsSuccess(message) {
+            // Display the alert.
+            let successAlert = $("#alert");
+            successAlert.removeClass("alert-danger").addClass("alert-success");
+
+            let alertContent = $("#alert-content");
+            alertContent.html(message); // Replace the existing message. TODO: queue alerts?
+
+            successAlert.show();
+            $(".tableFloatingHeaderOriginal").hide();
+
+            // TODO: document
+            successAlert.fadeTo(2000, 500).slideUp(500, () => {
+                successAlert.slideUp(500, () => {
+                    successAlert.removeClass("alert-success");
+                    alertContent.empty();
+                    $(".tableFloatingHeaderOriginal").show(); // hack until header bug is fixed (todo)
```

Comment:
This seems to be a frequent piece of code. Please add a bug and link to the tracking bug

Reply:
Should be rectified if we do #83